.circleci/config.yml

```diff
@@ -5,6 +5,7 @@ jobs:
         - image: circleci/node:13-browsers
           environment:
             CHROME_BIN: "/usr/bin/google-chrome"
+        - resource_class: medium+
```

Comment:
That's all it needed I guess. I guess I needed to look through the CircleCI docs instead of SO.

Reply:
We can't use medium+ anyway as it transpires, it's not available on the free tier.